tests/integration/targets/acme_letsencrypt/dns-challenge-missing-acme-domain.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Test role if acme_domain is not set
+  hosts: localhost
+  roles:
+    - telekom_mms.acme.acme
+  vars:
+    acme_challenge_provider: pebble
+    acme_use_live_directory: false
+    acme_account_email: ssl-admin@example.com
+    acme_staging_directory: https://localhost:14000/dir
+    acme_validate_certs: false
+  ignore_errors: true
```

Comment:
that way this will always succeed. We need to include some tasks at the end, that verify the response of the role.

Reply:
this still needs a additional check. Maybie via `register_result` and then `assert` in a next task.